src/popup/components/Modals/AccountCreate.vue

```diff
@@ -7,14 +7,15 @@
     @close="resolve"
   >
     <h2 class="text-heading-2">
-      {{ $t('modals.createAccount.title') }}
+      {{ isMultisig ? $t('modals.createAccount.titleMultisig') : $t('modals.createAccount.title') }}
```

Comment:
what do you think about splitting?

```suggestion
      {{
       isMultisig 
       ? $t('modals.createAccount.titleMultisig') 
       : $t('modals.createAccount.title') 
       }}
```